<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QwtPlot" name="plotSignal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="xSignal">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>40</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>X signal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ySignal">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Y signal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FFT">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>250</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>FFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IFFT">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>280</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IFFT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>390</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
